src/reconcile/rec_visibility.c

```diff
@@ -352,8 +368,13 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
         }
         if (upd != NULL) {
             /* The beginning of the validity window is the selected update's time pair. */
-            upd_select->start_ts = upd->start_ts;
-            upd_select->start_durable_ts = upd->durable_ts;
+            upd_select->start_durable_ts = upd_select->start_ts = upd->start_ts;
```

Comment:
Same here. Can you move the comment above this line?

Reply:
Done.